tools/common/Application.cs

```diff
@@ -490,19 +490,26 @@ public void RunRegistrar ()
 			ps.AssemblyResolver = resolver;
 			resolvedAssemblies.Add (ps.AssemblyResolver.Resolve (AssemblyNameReference.Parse ("mscorlib"), new ReaderParameters ()));
 
-			var rootName = Path.GetFileNameWithoutExtension (RootAssembly);
-			if (rootName != Driver.GetProductAssembly (this))
-				throw ErrorHelper.CreateError (66, "Invalid build registrar assembly: {0}", RootAssembly);
+			int index = 0;
+			foreach (var asm in RootAssemblies) {
+				var rootName = Path.GetFileNameWithoutExtension (asm);
+				if (index == 0 && rootName != Driver.GetProductAssembly (this))
+					throw ErrorHelper.CreateError (66, "Invalid build registrar assembly: {0}", RootAssembly);
```

Comment:
New errors need to be added to https://github.com/xamarin/xamarin-macios/blob/master/docs/website/mmp-errors.md

Reply:
This wasn't a new error, not even reworded. Will do after I change the previous request to add a new one.